prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html

```diff
@@ -54,6 +54,22 @@
                   [showIfEmpty]="true"
                   (route)="onRoute(demographicRoutePath)"></app-address-view>
 
+<ng-container *ngIf="enrollee?.additionalAddresses.length">
+  <app-page-subheader>
+    <ng-container appPageSubheaderTitle>Additional Addresses</ng-container>
+  </app-page-subheader>
+  <ng-container *ngFor="let additionalAddress of enrollee?.additionalAddresses; let index = index">
+    <app-page-subheader>
+      <ng-container appPageSubheaderSummary>
+        <strong> Address: {{ index + 1 }}</strong>
+      </ng-container>
+    </app-page-subheader>
+    <app-address-view [address]="additionalAddress"
+                      [showRedirect]="showEditRedirect"
+                      [showIfEmpty]="enrollee.gpid | contains: PAPER_ENROLLEE_GPID_PREFIX: 'startsWith'">
```

Comment:
This logic is no longer needed, since we only show if the address exists in the for